designs/CentrallyManagingNuGetPackageVersions-VSManagePackages.md

```diff
@@ -0,0 +1,108 @@
+# Centrally managing NuGet package versions - Managing packages in Visual Studio Spec
+
+* Status: **In Review**
+* Author(s): [Andy Zivkovic](https://github.com/zivkan)
+
+## Related Specs
+
+* [Centrally managing NuGet package versions](https://github.com/NuGet/Home/wiki/Centrally-managing-NuGet-package-versions)
+* [Centrally managing NuGet package versions - Restore Design Spec](https://github.com/NuGet/Home/blob/dev/designs/CentrallyManagingNuGetPackageVersions-Restore.md)
+
+## Problem Background
+
+To date, NuGet does not modify any MSBuild files (projects files, or files imported by the projects) either directly or though MSBuild APIs. Instead, NuGet uses APIs provided by the various project systems when projects need to be modified as a result of installing, updating or removing packages. In practise, NuGet has only indirectly caused project files to be modified.
```

Comment:
All of this is a VS design principle afaik. If you want to modify the project file you go through the project system.